### PR TITLE
Updated some spec and other files

### PR DESCRIPTION
**Pull request by bwelch8105**
Updated some spec and other files to fix and/or skip tests that are currently failing.

### CODE REVIEW

File: cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts [hunk 1]
```diff
@@ -68,7 +66,7 @@ describe('Test Case Validations', () => {
 
     })
 
-    it('Create Test Case: Description more than 250 characters', () => {
+    it.skip('Create Test Case: Description more than 250 characters', () => {
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts:69]: why are we skipping this test?

**Review by cvasile001 — CHANGES_REQUESTED** [review]
I think we are missing a comment for one of those skipped tests

File: cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts (hunk 1 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts]: Due to the bug where the login page does not load, I would get one or two tests to run, at a time, before hitting that bug. So, to help move things along a little quicker, I would skip all of the tests that passed with each attempt to run the spec file. That means by the time I got to the end of the spec file, I had to go back and remove a lot of .skip. I must have overlooked this one, when I removed all of the ones that were not necessary.

I have removed it now, though.

**Review by cvasile001 — APPROVED** [review]
approved

**Pull request merged**